### PR TITLE
Add unit tests

### PR DESCRIPTION
**Pull request by Sakshi-75**
This is for the issue #17 
I've added unit tests for the class CMLAbundance.java
I've also covered the methods that have been inherited in it from AbstractAbundance.java

### CODE REVIEW

**Comment by Sakshi-75** [comment]
Hi @egonw - I have added unit test case for CMLAbundance.java file. Kindly review it and provide feedbacks if any. Thanks!

File: src/test/java/org/xmlcml/cml/element/main/CMLAbundanceTest.java [hunk 1]
```diff
@@ -0,0 +1,408 @@
+/**
+ *    Copyright 2011 Peter Murray-Rust et. al.
```
Comment (egonw) [inline src/test/java/org/xmlcml/cml/element/main/CMLAbundanceTest.java]: This should state your own name, ie. of the person who wrote the code. Also, the year should be the year when you wrote the code.

Comment (Sakshi-75) [reply src/test/java/org/xmlcml/cml/element/main/CMLAbundanceTest.java]: Fixed

**Review by egonw — APPROVED** [review]
Looks good.

**Pull request merged**